storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -62,4 +62,7 @@
     @NonNull
     @CheckResult
     Single<Result> asRxSingle();
+
+    @NonNull
+    Object getData();
```

Comment:
Generics impossible for `PreparedGetOperation`, cause it can have either `Query` or `RawQuery`. We can create common interface for them, but what a point? Anyway, it will not add any details, cause user will see common generic parameter inside `Interceptor` which is sane as seeing `Object`.